### ISSUE TITLE
UI are different in MacVim

### ISSUE THREAD

**Issue by markwu** [liuchengxu/vim-clap#231]
**Environment (please complete the following information):**
 - OS: macOS 10.14.6
 - (Neo)Vim version: NeoVim git master head
 - vim: vim git master head
 - macvim: macvim git master head
 - vim-clap version: vim-clap git master head

**Describe the bug**
UI are different in MacVim

**To Reproduce steps**

Open clap with `:Clap files`, and type test, and down 3 times and up 3 times.

**neovim**
![vim-clap-1](https://user-images.githubusercontent.com/196861/72220223-81a3aa80-3589-11ea-932a-e0e112ecc14c.gif)

**vim**
![vim-clap-2](https://user-images.githubusercontent.com/196861/72220224-86685e80-3589-11ea-9b73-aa4eeb31fbb4.gif)

**macvim (gui version)**
![vim-clap-3](https://user-images.githubusercontent.com/196861/72220226-89fbe580-3589-11ea-9e23-7c12cd20f074.gif)

`Indicator` and `highlight bar` just disappear. `echo has('patch-8.1.2114')` is return `1`.

**My minimal vimrc**
```vim
set nocompatible
filetype off

execute 'source ~/.vim/bundle/vim-plug/plug.vim'
call plug#begin('~/.vim/bundle/')

Plug 'junegunn/vim-plug'

" Put your bundle below here
Plug 'liuchengxu/vim-clap'
" Put your bundle above here

call plug#end()

syntax on
filetype plugin indent on

" Put your plugin settings below here

" Put your plugin settings above here
```

**Expected behavior**
Should be looks the same.

**Comment by liuchengxu** [comment]
Can not reproduce. Try a new colorscheme to see if you can see the indicator.

![image](https://user-images.githubusercontent.com/8850248/72220701-5b343e00-358e-11ea-929a-648d3f8f7320.png)

**Comment by markwu** [comment]
It still the same when I change to different color scheme.  Weird.

![螢幕快照 2020-01-12 下午11 53 51](https://user-images.githubusercontent.com/196861/72221555-0648f580-3597-11ea-9a83-189d2a083f9b.png)
![螢幕快照 2020-01-12 下午11 54 21](https://user-images.githubusercontent.com/196861/72221556-077a2280-3597-11ea-88fe-57d0e0b9d1ed.png)

**Comment by liuchengxu** [comment]
I figured it out. There was a similar issue https://github.com/liuchengxu/vim-clap/issues/130, the fix needs 8.1.2362,  but MacVim onlys has 8.1.2318 right now. You should report to MacVim for the upgrade then.

**Comment by markwu** [comment]
Ah ... I see, Thanks!
